luthier/src/test/groovy/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialParkTest.groovy

```diff
@@ -0,0 +1,25 @@
+package com.yahoo.bard.webservice.config.luthier
+
+import com.yahoo.bard.webservice.data.config.ResourceDictionaries
+import com.yahoo.bard.webservice.data.dimension.Dimension
+import spock.lang.Specification
+
+class LuthierIndustrialParkTest extends Specification {
+    LuthierIndustrialPark industrialPark
+    void setup() {
+        ResourceDictionaries resourceDictionaries = new ResourceDictionaries()
+        Map<String, Factory<Dimension>> dimensionFactoriesMap = new HashMap<>()
+        industrialPark = new LuthierIndustrialPark(resourceDictionaries, dimensionFactoriesMap)
```

Comment:
I would use the Builder rather than building a LuthierIndustrialPark with the constructor:
```groovy
industrialPark = new LuthierIndustrialPark.Builder()
.withDimensionFactory("test", new TestDimensionFactory())
.build()
```

Reply:
Fixed.